packages/frontend/src/components/profile/remove_account/RemoveAccountWrapper.js

```diff
@@ -11,8 +12,28 @@ import Container from '../../common/styled/Container.css';
 import RemoveAccountModal from './RemoveAccountModal';
 
 const StyledContainer = styled(Container)`
+    margin-top: 16px;
+    padding-top: 0;
+    padding-bottom: 0;
     > button {
         width: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        > svg {
+            width: 22px !important;
```

Comment:
nit: can we please opt to use `&` (if absolutely needed) for added specificity instead of `!important`